packages/edit-site/src/components/sidebar/global-styles/preview.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Card,
+	ColorIndicator,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */ import { useGlobalStylesContext } from '../../editor/global-styles-provider';
+
+export const StylePreview = () => {
+	const { getStyle } = useGlobalStylesContext();
+	const fontFamily = getStyle( 'root', 'fontFamily' ) ?? 'serif';
+	const fontSize = getStyle( 'root', 'fontSize' ) ?? 'initial';
```

Comment:
This one should be set by us: in figma it's at 80px with a box of 248x152px (on an 8px grid).